### PR TITLE
[Docs] Improve doc for Elasticsearch authentication

### PR DESCRIPTION
**Pull request by li-boxuan**
-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: docs/configs/janusgraph-cfg.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ Configuration options for HTTP(S) authentication.
 
 | Name | Description | Datatype | Default Value | Mutability |
 | ---- | ---- | ---- | ---- | ---- |
-| index.[X].elasticsearch.http.auth.type | Authentication type to be used for HTTP(S) access. | String | NONE | LOCAL |
+| index.[X].elasticsearch.http.auth.type | Authentication type to be used for HTTP(S) access. Default value is `NONE`. Other options are `BASIC` and `CUSTOM`. | String | NONE | LOCAL |
```
Comment (farodin91) [inline docs/configs/janusgraph-cfg.md:155]: Do we really have to say the default value?

Comment (li-boxuan) [reply docs/configs/janusgraph-cfg.md:155]: The wording here can be improved. Do you think it's better if we say "Available options are `NONE`, `BASIC` and `CUSTOM`"?

Comment (farodin91) [reply docs/configs/janusgraph-cfg.md:155]: Sounds better

**Review by farodin91 — APPROVED** [review]
LGTM! Thank you @li-boxuan.

**Pull request merged**